src/redux/rockets/reducer.js

```diff
@@ -0,0 +1,38 @@
+import axios from 'axios';
+import * as actions from './actions';
+
+const stateInit = [];
+const url = axios.create({ baseURL: 'https://api.spacexdata.com/v3/rockets' });
```

Comment:
This works pretty well, but if we want to keep our code DRY, we should extract our Axios config to a separate file. and reuse it in all of our reducers.

Reply:
**Update**: I already created an Axios file in my own branch, you can use it if you want.